packages/home-service/src/resolver.ts

```diff
@@ -1,31 +1,45 @@
 import { Home } from './schema';
-import { pubsub } from './helpers';
-
-const Home_UPDATE = 'Home_UPDATE';
 
 export const HomeResolver = {
+  // queries
```

Comment:
Unnecessary comment. Does not serve any purpose, as it is exactly the same as the key name below it.

@riginoommen Might want to remove it from the cli template as well.

Reply:
@deshmukhmayur Unnecessary comments removed from the document